README.md

```diff
@@ -3,7 +3,10 @@
 [![MongoDB Atlas Go Client](https://img.shields.io/badge/Powered%20by%20-go--client--mongodb--atlas-%2313AA52)](https://github.com/mongodb/go-client-mongodb-atlas)
 
 Welcome to MongoDB Atlas Operator - the Operator allowing to manage Atlas Clusters from Kubernetes.
-> Current Status: *pre-alpha*. We are currently working on the initial set of features that will give users the opportunity to provision Atlas projects, clusters and database users using Kubernetes Specifications and bind connection information into the applications deployed to Kubernetes.
+> Current Status: *alpha*. We are currently working on the initial set of features that will give users the opportunity 
```

Comment:
have changed "pre-alpha" to "alpha".
Also added a couple of new lines into the paragraph